pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -264,6 +256,14 @@ func (ks *scaler) handleScaleToZero(ctx context.Context, pa *pav1alpha1.PodAutos
 			logger.Info("Probe for revision is already in flight")
 		}
 		return desiredScale, false
+	default: // Active=Unknown
```

Comment:
Can we order them the old way? I.e. just check for `IsActivating`?

Reply:
Ack, seems like we could just keep the ordering now :thinking:

Reply:
I think it reads better as "active, not active, activating" and activating makes for a better default case if something is missed in the future, but I've reverted the ordering.

Reply:
It's the way the scale down state machine kind of works. If you come from this point of view it makes more sense.